input/pages/09-b-cqlreference.md

```diff
@@ -6092,7 +6095,7 @@ valueset "Value Set Reference": 'http://example.org/ValueSet/ExampleValueSet'
 
 define TestExpandValueSet: ExpandValueSet("Value Set Reference")
 ```
-
+{: .contains_valueset}
```

Comment:
There is already an anchor here from the header, no need for this change:
https://cql.hl7.org/2025Sep/09-b-cqlreference.html#contains-valueset